easybuild/easyblocks/i/itac.py

```diff
@@ -62,11 +63,18 @@ def install_step(self):
         """
 
         if LooseVersion(self.version) >= LooseVersion('8.1'):
-            super(EB_itac, self).install_step(silent_cfg_names_map=None)
+            silent_cfg_names_map = {}
+            if LooseVersion(self.version) >= LooseVersion('9.0'):
+                # To generate the correct itacvars.sh (called by traceanalyzer) we need to change the default install dir
+                silent_cfg_names_map = {
+                    'install_dir': install_path(),
```

Comment:
Well, we can also simply not do `move_after_install`, and accept the silly double subdirectory .

I agree it isn't right, but it's not EB's fault imho, it's the `itac` installer.